src/app/component/CheckoutShipping/CheckoutShipping.container.js

```diff
@@ -64,16 +66,20 @@ export class CheckoutShippingContainer extends PureComponent {
     }
 
     onShippingSuccess(fields) {
-        const { saveAddressInformation } = this.props;
+        const { saveAddressInformation, addressLinesQty } = this.props;
 
         const {
             selectedCustomerAddressId,
             selectedShippingMethod
         } = this.state;
 
+        const formFields = addressLinesQty > 1
```

Comment:
I believe this logic can be hidden in "getFormFields" function with the only responsibility of handling this logic.
Please use this approach in order to get rid of the ternary operator and to make code more readable.